README.md

```diff
@@ -50,33 +49,63 @@ docker run -p 80:80 -p <managedurl_port>:8080 -p 9000:9000 \
 ```
 
 ## API
-- [v2 Management Interface](https://github.com/openwhisk/openwhisk-apigateway/blob/master/doc/v2/management_interface_v2.md)
-- [v1 Management Interface](https://github.com/openwhisk/openwhisk-apigateway/blob/master/doc/v1/management_interface_v1.md)
 
+* [v2 Management Interface](https://github.com/openwhisk/openwhisk-apigateway/blob/master/doc/v2/management_interface_v2.md)
+* [v1 Management Interface](https://github.com/openwhisk/openwhisk-apigateway/blob/master/doc/v1/management_interface_v1.md)
 
 ## Developer Guide
 
 ### Running locally
 
  To build the docker image locally use:
+
  ```
   make docker
  ```
 
  To Run the Docker image
+
  ```
-  make docker-run PUBLIC_MANAGEDURL_HOST=<mangedurl_host> PUBLIC_MANAGEDURL_PORT=<managedurl_port> \
-    REDIS_HOST=<redis_host> REDIS_PORT=<redis_port> REDIS_PASS=<redis_pass>
+ make docker-run \
+    PUBLIC_MANAGEDURL_HOST=<mangedurl_host> \
+    PUBLIC_MANAGEDURL_PORT=<managedurl_port> \
+    REDIS_HOST=<redis_host> REDIS_PORT=<redis_port> \
+    REDIS_PASS=<redis_pass>
  ```
 
-
 ### Testing
 
  First install the necessary dependencies:
+
  ```
   make test-build
  ```
+
  Then, run the unit tests:
+
  ```
   make test-run
  ```
+
+### Multi-architecture build
+
+ The API Gateway has been converted to support multi-architecture builds for
+ the standard (non-profiling) build. As a result, all builds are now managed
+ by the Gradle build tool. For backward compatibility, the `Makefile` can be
+ used to execute the builds and runs as always.
+
+ For more information on building a multiarchitecture OpenWhisk image, see
+ [this document](https://github.com/apache/incubator-openwhisk/blob/master/docs/runtimes-multiarch.md).
```

Comment:
broken link ?

Reply:
File is being created in https://github.com/apache/incubator-openwhisk/pull/3496 , which will hopefully go first.